simulator/src/main/resources/app.properties

```diff
@@ -33,8 +33,8 @@ prometheus.endpointPortNumber=9998
 
 #blockStream.maxBlockItemsToStream=100_000_000
 #blockStream.streamingMode=MILLIS_PER_BLOCK
-#blockStream.millisecondsPerBlock=500
-#blockStream.blockItemsBatchSize=1_000
+blockStream.millisecondsPerBlock=500
+blockStream.blockItemsBatchSize=1_000
```

Comment:
maybe drop these defaults, I think you changed these during development of the new feature, but by default they should be commented out?

Reply:
you are right, changes to the properties here were unintentional.

✅ Done!